### DISCUSSION TITLE
cddb / cdindex support?

### DISCUSSION THREAD

**Discussion by Vrihub** [beetbox/beets#4231] [Ideas]
I've just ripped a CD using the icedax tool, which created a `.cdindex` XML file getting metadata from the CDDB public online database. I expected beets to be able to use this info in the import stage, perhaps via a plugin, but it seems like CDDB is not supported. Is this a deliberate choice (given the problems with CDDB)? Or simply noone thought about it yet?

This particular CD is contained in the CDDB database, but not in musicbrainz, and since musicbrainz has removed the possibility to import data from CDDB (see https://tickets.metabrainz.org/browse/MBS-9143), a beets plugin to pull data directly from CDDB could be useful. In your experience, is this a common occurrence? Or did I just stumble into an unlucky CD?

**Comment by wisp3rwind** [comment]
Searching the Github issues, this has simply not been brought up. beets is originally very musicbrainz-centric, so that's probably not too surprising. I guess this could be added in principle. On the other hand, https://gnudb.org/ seems to contain generally very little information compared to MB or discogs, so the usefulness might be very limited, in particular if your ripper already tagged the files from some CDDB provider.

Ultimately, this will depend on somebody (...) actually doing the work.
